api/v1alpha1/virtualmachinepublishrequest_types.go

```diff
@@ -278,7 +339,7 @@ type VirtualMachinePublishRequestStatus struct {
 	Conditions []Condition `json:"conditions,omitempty"`
 }
 
-func (vmpr VirtualMachinePublishRequest) getCondition(
+func (vmpr VirtualMachinePublishRequest) GetCondition(
```

Comment:
No we can't use the internal function since VMPub doesn't implement `Getter` interface. :( This vmPubReq has its own way to update and check conditions. Implement Getter interface seems redundant.